pypeit/spectrographs/bok_bc.py

```diff
@@ -313,9 +313,12 @@ def config_specific_par(self, scifile, inp_par=None):
         # Start with instrument wide
         par = super().config_specific_par(scifile, inp_par=inp_par)
 
+        headarr = self.get_headarr(scifile)
+        
         # Wavelength calibrations
-        #if self.get_meta_value(scifile, 'dispname') == 'R1200B':
-        #    par['calibrations']['wavelengths']['reid_arxiv'] = 'wht_isis_blue_1200_4800.fits'
+        if self.get_meta_value(scifile, 'dispname') == '300':
```

Comment:
Do we need the `headarr = self.get_headarr(scifile)` since we're grabbing the grating name straight from `scifile`?